master/lustrefactory.py

```diff
@@ -56,6 +62,65 @@ def buildzfsCommand(props):
     return args
 
 @util.renderer
+def configureCmd(props):
+    args = ["./configure"]
+    style = props.getProperty('buildstyle')
+
+    if style == "deb" or style == "rpm":
```

Comment:
I don't anticipate many (if any) more conditionals to be added. All the major cases are covered (make srpm then rpmbuild, make debs, make rpms, and a simple make). So, this should remain static. If for some reason many more cases are suddenly necessary, it can always be broken out into a factory per platform.